### PR TITLE
ST110 - Report, Trigger, Widget, Wizard, XDRCTemplate and XSIAMDashboard

### PR DESCRIPTION
**Pull request by rshunim**
## Related Issues
relates: https://jira-dc.paloaltonetworks.com/browse/CIAC-7809

## Description
Added Report, Trigger, Widget, Wizard, XDRCTemplate and XSIAMDashboard objects of Pydantic for ST110 validation.

### CODE REVIEW

File: demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py [hunk 1]
```diff
@@ -177,3 +177,30 @@ class _StrictGenericIncidentType(BaseStrictModel):
         BaseOptionalVersionJson,
     ),
 )
+
+
+SCHEMAS = Union["Filter", "Or", "And"]
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py:182]:
```suggestion
OPERATORS = Union["Filter", "Or", "And"]
```

File: demisto_sdk/commands/content_graph/strict_objects/widget.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    is_predefined: bool = Field(alias="isPredefined")
+    date_range: Optional[Dict[Any, Any]] = Field(None, alias="dateRange")
+    params: Optional[Dict[Any, Any]] = None
+    size: Optional[int] = None
+    sort: Optional[List[Any]] = None
+    category: Optional[str] = None
+    modified: Optional[str] = None
+    marketplaces: Optional[List[str]] = Field(
+        None, enum=["xsoar", "xsoar_saas", "xsoar_on_prem"]
+    )
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/widget.py:34]:
```suggestion
    marketplaces: Optional[List[MarketplaceVersions]] = None
```

File: demisto_sdk/commands/content_graph/strict_objects/widget.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+class _StrictWidget(BaseStrictModel):
+    id_: str = Field(alias="id")
+    version: int
+    name: str
+    description: str
+    data_type: Optional[str] = Field(None, alias="dataType")
+    widget_type: str = Field(alias="widgetType")
+    query: Optional[str] = None
+    is_predefined: bool = Field(alias="isPredefined")
+    date_range: Optional[Dict[Any, Any]] = Field(None, alias="dateRange")
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/widget.py:26]: can we model this one too?

File: demisto_sdk/commands/content_graph/strict_objects/widget.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    params: Optional[Dict[Any, Any]] = None
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/strict_objects/widget.py:27]: can we model this one too?

File: demisto_sdk/commands/content_graph/strict_objects/widget.py (hunk 3 above)
Comment (rshunim) [reply demisto_sdk/commands/content_graph/strict_objects/widget.py:26]: There are no examples in Content

File: demisto_sdk/commands/content_graph/strict_objects/widget.py (hunk 4 above)
Comment (rshunim) [reply demisto_sdk/commands/content_graph/strict_objects/widget.py:27]: Here too

**Review by dorschw — APPROVED** [review]

**Pull request merged**